### DISCUSSION TITLE
Parsing Strings c++

### DISCUSSION THREAD

**Discussion by Corey-Thompson-88** [Corey-Thompson-88/Design-Patterns#14] [Q&A] [answered]
How do I parse a binary string?

**Comment by Tanner-Davison** [comment, marked as answer]
std::string binaryString = "1010";
std::bitset<4> bits(binaryString);  // specify size of 4 bits
int decimal = (int)bits.to_ulong(); 

frist example I will meet with you after and we can talk about it there.

**Comment by Tanner-Davison** [comment]
also 
int num = 10;
std::string binary = std::bitset<4>(num).to_string();
